vic/drivers/shared_image/include/vic_driver_shared_image.h

```diff
@@ -27,6 +27,7 @@
 #ifndef VIC_DRIVER_SHARED_IMAGE_H
 #define VIC_DRIVER_SHARED_IMAGE_H
 
+#include <image_extension_name.h>
```

Comment:
should we call this `rout_extension_name.h`?

Reply:
This indeed streamlines better with @bartnijssen latest additions. 
Done.